kit/Kit.cpp

```diff
@@ -3190,12 +3221,23 @@ void lokit_main(
             }
 
 #ifndef __FreeBSD__
-            dropCapability(CAP_SYS_CHROOT);
-            dropCapability(CAP_FOWNER);
-            dropCapability(CAP_CHOWN);
+            if (usingMountNamespace)
```

Comment:
Cap dropping is tricky and seems to be copy/pasted below (?) =)

Reply:
can certainly bundle these together inside a utility